src/constants.js

```diff
@@ -192,6 +192,13 @@ export const TASKS_PAGE_HEADER = [
  * Table constants
  */
 
+export const COMPLETED_TASKS_TABLE_DEFAULTS = {
```

Comment:
In the future this should be refactored so that its not just a hard coded integer.